docs/en/multi_modal/minicpmv.md

```diff
@@ -1,24 +1,208 @@
+|        Model         | Supported Inference Engine |
+| :------------------: | :------------------------: |
+| MiniCPM-Llama3-V-2_5 |         TurboMind          |
+|    MiniCPM-V-2_6     |         TurboMind          |
 
-## Quick Start
+The next chapter demonstrates how to deploy an MiniCPM-V model using LMDeploy, with [MiniCPM-V-2_6](https://huggingface.co/openbmb/MiniCPM-V-2_6) as an example.
 
-Please install LMDeploy by following the [installation guide](../installation.md)
+## Installation
 
-### Offline inference pipeline
+Please install LMDeploy by following the [installation guide](../installation.md).
 
-The following sample code shows the basic usage of VLM pipeline. For more examples, please refer to [VLM Offline Inference Pipeline](./vl_pipeline.md)
+## Offline inference
+
+The following sample code shows the basic usage of VLM pipeline. For detailed information, please refer to [VLM Offline Inference Pipeline](./vl_pipeline.md)
 
 ```python
 from lmdeploy import pipeline
 from lmdeploy.vl import load_image
 
-pipe = pipeline('openbmb/MiniCPM-Llama3-V-2_5')
+pipe = pipeline('openbmb/MiniCPM-V-2_6')
 
 image = load_image('https://raw.githubusercontent.com/open-mmlab/mmdeploy/main/tests/data/tiger.jpeg')
 response = pipe(('describe this image', image))
 print(response)
 ```
+
+More examples are listed below:
+
+<details>
+  <summary>
+    <b>Chat with multiple images</b>
+  </summary>
+
+```python
+from lmdeploy import pipeline, GenerationConfig
+
+pipe = pipeline('openbmb/MiniCPM-V-2_6', log_level='INFO')
+messages = [
+    dict(role='user', content=[
+        dict(type='text', text='Describe the two images in detail.'),
+        dict(type='image_url', image_url=dict(max_slice_nums=9, url='https://raw.githubusercontent.com/OpenGVLab/InternVL/main/internvl_chat/examples/image1.jpg')),
+        dict(type='image_url', image_url=dict(max_slice_nums=9, url='https://raw.githubusercontent.com/OpenGVLab/InternVL/main/internvl_chat/examples/image2.jpg'))
+    ])
+]
+out = pipe(messages, gen_config=GenerationConfig(top_k=1))
+print(out.text)
+
+messages.append(dict(role='assistant', content=out.text))
+messages.append(dict(role='user', content='What are the similarities and differences between these two images.'))
+out = pipe(messages, gen_config=GenerationConfig(top_k=1))
+print(out.text)
+```
+
+</details>
+
+<details>
+  <summary>
+    <b>In-context few-shot learning</b>
+  </summary>
+
+```python
+from lmdeploy import pipeline, GenerationConfig
+
+pipe = pipeline('openbmb/MiniCPM-V-2_6', log_level='INFO')
+
+question = "production date"
+messages = [
+    dict(role='user', content=[
+        dict(type='text', text=question),
+        dict(type='image_url', image_url=dict(url='example1.jpg')),
```

Comment:
what are the two images?

Reply:
copyied from https://github.com/OpenBMB/MiniCPM-V?tab=readme-ov-file#in-context-few-shot-learning and I can't find there images in their repo.